src/settings/toml/tests/deploy_target.rs

```diff
@@ -619,7 +661,8 @@ fn when_top_level_zoneless_env_zoned_single_route() {
     let env_config = EnvConfig::zoned_single_route(zone_id, pattern);
 
     let script = "top_level_zoneless_env_zoned_single_route_no_zone_id";
-    let test_toml = WranglerToml::webpack_zoneless_with_env(script, true, env_name, env_config);
+    let test_toml =
```

Comment:
i wonder if it makes sense to just create a struct every time instead of positional parameters - it's kind of hard to read

Reply:
discussed this in air also; creating a struct every time would require that you declare the entire toml, in which case we may as well just go back to writing out entire toml strings. however as a compromise we can reduce the number of positional arguments in some of these